docs/powervs-faq.md

```diff
@@ -0,0 +1,24 @@
+# PowerVS Frequently Asked Questions
+
+---
+## When Building out a Cluster
+
+#### VM Console shows "grub>"
+Not the end of the world. Just ESC and ENTER, as the message on screen instructs.
+
+#### RHCOS Ignition stalls with "version mismatch"
+The version of RHCOS you are booting up with on cluster nodes cannot handle what the OCP build/version is serving as config for cluster nodes. Make sure they match at least the major and minor versions (eg. 4.5) in your `var.tfvars` file.
+
+---
+## Problems on a Running Cluster
+
+#### Need to add more CPU/Memory to Node
+Go to <https://cloud.ibm.com>, then to your PowerVS service instance, find your VM that backs the cluster Node (eg. myocp-worker-0), click to open up the details page, then "Edit details," update the CPU/Memory with new values, and Save. You should get "Edit successful" message, and your OCP cluster should recognize the change and start showing in Compute section of Web Console. If something fails along the way, try "OS Shutdown" action, edit CPU/Memory as necessary while in "Shutoff" state, and restart the VM.
+
+#### Pods started failing on a specific Node
+If you see multiple Pods failing on only 1 node of your cluster, suspect "ephemeral storage full" situation. See the relevant entry in [Extenral links page](../actually-helped#2020-10).
+
```

Comment:
OCP should give "DiskPressure" event if ephemeral storage gets full. This is a generic issue on any platform.

Reply:
Understood this is a generic issue. Do you not want to have this?

Reply:
Lets drop this since it's not PowerVS specific.  This is more suited for k8s or ocp faqs.

Reply:
Fine, dropped in the latest commit.